filters/ThirdParty/filter_233_AdblockListFinland/metadata.json

```diff
@@ -1,17 +1,17 @@
 {
   "filterId": 233,
-  "name": "Juvander's Adblock List for Finland",
+  "name": "Adblock List for Finland",
   "description": "Finnish adblock list",
   "timeAdded": 1404115015843,
-  "homepage": "https://www.juvander.fi/ilpo",
+  "homepage": "https://github.com/theel0ja/finnish-easylist-addition",
   "expires": "2 days",
   "displayNumber": 2,
   "groupId": 7,
-  "subscriptionUrl": "http://adb.juvander.net/Finland_adb.txt",
+  "subscriptionUrl": "https://adb.theel0ja.info/finnish-easylist-addition/Finland_adb.txt",
   "tags": [
     "purpose:ads",
     "recommended",
     "lang:fi"
   ],
   "disabled": true
```

Comment:
If it's safe to use it now, please remove this property

Reply:
we can remove exclusions, that rules are not exit in this filter